pkgs/tools/graphics/realesrgan-ncnn-vulkan/default.nix

```diff
@@ -50,15 +51,19 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin $out/share
+
     cp realesrgan-ncnn-vulkan $out/bin/
+    wrapProgram $out/bin/realesrgan-ncnn-vulkan \
+      --prefix LD_LIBRARY_PATH ':' "${lib.makeLibraryPath [ vulkan-loader ]}"
```

Comment:
Can we use patchelf instead? LD_LIBRARY_PATH has many disadvantages and taints spawned programs from the binary.